vignettes/articles/introduction.Rmd

```diff
@@ -220,7 +220,7 @@ See https://docs.lamin.ai/tutorial#query-search-registries
 
 ```{r query-registries}
 ln$Artifact$to_dataframe()
-ln$Artifact$to_dataframe(features = TRUE)
+ln$Artifact$to_dataframe(include = "features")
```

Comment:
This is the newer way of doing it, but had nothing to do with the actual test failure.

Reply:
(The old syntax is still supported.)